### ISSUE TITLE
Support for ppc64le

### ISSUE THREAD

**Issue by jjhursey** [tigera/operator#1842]
I'm trying to install Calico on a `ppc64le` machine, which it says that it supports [here](https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart). However, there is not a `ppc64le` container image of `quay.io/tigera/operator:v1.25.3` that I can pull down. Only `amd64` and `arm64`.

## Expected Behavior

Following the [Calico Quickstart Guide](https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart) on a Power8 machine the following should create the operator on the system .

```
kubectl create -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
```

## Current Behavior

Following the [Calico Quickstart Guide](https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart) on a Power8 machine the following will fail in `ImagePullBackOff` because of the lack of an image.

```
kubectl create -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
```

```
$ kubectl get pods --all-namespaces | grep tigera
tigera-operator   tigera-operator-b876f5799-9lxgf   0/1     ImagePullBackOff   0          17m
```

## Possible Solution

Add a build for `ppc64le`

## Steps to Reproduce (for bugs)

1. `kubectl create -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml`

## Context

I'm trying to install Calico on a new Kubernetes environment installed via `kubeadm`

## Your Environment

* Operating System and version: RHEL 8.4 (ppc64le)

**Comment by tmjd** [comment]
@caseydavenport I don't think we considered the architectures besides amd64 and arm when the quickstart was switch to operator. I'm guessing it isn't too much work to add those other archs but I'm not sure when this work would get prioritized.
@jjhursey if you would want to add support I would be happy to review. :grinning:

**Comment by jjhursey** [comment]
I ended up following [this](https://projectcalico.docs.tigera.io/getting-started/kubernetes/self-managed-onprem/onpremises#next-steps) guide to deploy Calico without `Tigera` for now.

I can take a look to see how difficult it would be to add the `ppc64le` build to your mix. I took a quick look at the `Makefile` and it doesn't seem to bad. I'm not sure when I would get to it though.

**Comment by caseydavenport** [comment]
Quick PR that should fix this here: https://github.com/tigera/operator/pull/1855
